packages/babel-preset-typescript/src/index.ts

```diff
@@ -65,7 +66,13 @@ export default declarePreset((api, opts: Options) => {
           {
             test: /\.cts$/,
             sourceType: "script",
-            plugins: [[transformTypeScript, pluginOptions(false, true)]],
+            parserOpts: {
+              allowImportExportEverywhere: true,
+            },
```

Comment:
Can we change it here to use `"unambiguous"` instead of `allowImportExportEverywhere`?

Reply:
finished! I just forgot about this.